CSS/components/grids.css

```diff
@@ -0,0 +1,17 @@
+.grid-wrap-2-2{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: repeat(2, 1fr) ;
+    grid-template-columns:  repeat(2, 1fr);
+}
+.grid-wrap-3-3{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: repeat(3, 1fr);
+    grid-template-columns:  repeat(3, 1fr);
+}
+.grid-pills{
+    display: flex;
+    justify-content: center;
```

Comment:
You might wanna give it a standard name, grid-pills seems confusing. Also try implementing base classes.

Reply:
thank you for pointing out, done added a util class remove this whole class.